src/widget/wtracktableview.cpp

```diff
@@ -447,6 +447,15 @@ void WTrackTableView::slotPurge() {
     }
 }
 
+void WTrackTableView::slotDeleteTracksFromDisk() {
+    QModelIndexList indices = selectionModel()->selectedRows();
+
+    if (indices.size() > 0) {
+        m_pTrackMenu->loadTrackModelIndices(indices);
```

Comment:
Why do you call loadTrackModelIndices() again? This is only called once when opening the context menu.

Reply:
Without it `m_trackIndexList` wouldn't be populated because when pressing the Delete button in `DlgHidden` call to `WTrackTableView::contextMenuEvent`.

Reply:
..which would show an empty file list.
I could add a check for that to `WTrackMenu::slotRemoveFromDisk()` but actually it's not necessary, is it?